### PR TITLE
Refactor calls

### PR DESCRIPTION
**Pull request by In-line**

### CODE REVIEW

File: rehlds/engine/net_ws.cpp [hunk 1]
```diff
@@ -1023,14 +1023,14 @@ qboolean NET_QueuePacket(netsrc_t sock)
 
 	if (ret == -1 || ret == MAX_UDP_PACKET)
 	{
-		return NET_LagPacket(0, sock, 0, 0);
+		return NET_LagPacket(FALSE, sock, NULL, NULL);
```
Comment (WPMGPRoSToTeMa) [inline rehlds/engine/net_ws.cpp]: Should be `nullptr`, not `NULL`.

Comment (In-line) [inline rehlds/engine/net_ws.cpp]: @WPMGPRoSToTeMa Such changes should be made on a larger scale

**Comment by LevShisterov** [comment]
Most of the code use NULL.

File: rehlds/engine/net_ws.cpp (hunk 1 above)
Comment (WPMGPRoSToTeMa) [inline rehlds/engine/net_ws.cpp]: @In-line but better change `0` directly to `nullptr` (skip `NULL` stage).

Comment (In-line) [inline rehlds/engine/net_ws.cpp]: @WPMGPRoSToTeMa we can define NULL as nullptr

Comment (WPMGPRoSToTeMa) [inline rehlds/engine/net_ws.cpp]: @In-line better to replace all of them in another refactor commit.

**Pull request merged**